examples/getting_started/pt/swarm_script_executor_cifar10.py

```diff
@@ -47,19 +47,22 @@
         executor = ScriptExecutor(task_script_path=train_script)
         job.to(executor, f"site-{i}", gpu=0, tasks=["train", "validate", "submit_model"])
 
-        client_controller = SwarmClientController()
-        job.to(client_controller, f"site-{i}", tasks=["swarm_*"])
-
-        client_controller = CrossSiteEvalClientController()
-        job.to(client_controller, f"site-{i}", tasks=["cse_*"])
-
         # In swarm learning, each client acts also as an aggregator
         aggregator = InTimeAccumulateWeightedAggregator(expected_data_kind=DataKind.WEIGHTS)
         job.to(aggregator, f"site-{i}")
 
         # In swarm learning, each client uses a model persistor and shareable_generator
-        job.to(PTFileModelPersistor(model=Net()), f"site-{i}")
-        job.to(SimpleModelShareableGenerator(), f"site-{i}")
+        persistor = PTFileModelPersistor(model=Net())
+        job.to(persistor, f"site-{i}")
+        job.to(SimpleModelShareableGenerator(), f"site-{i}", id="shareable_generator")
```

Comment:
Shouldn't this also be set via `as_id()` in `SwarmClientController`. It has an `shareable_generator_id` argument.

Reply:
changed.